crates/cxx-qt-lib/src/gui/qpainter.rs

```diff
@@ -123,6 +125,9 @@ mod ffi {
         /// Returns the opacity of the painter. The default value is 1.
         fn opacity(self: &QPainter) -> f64;
 
+        /// Returns the painter's current pen.
+        fn pen(self: &QPainter) -> &QPen;
```

Comment:
hmm what about `setPen(const QPen&)` we need some way of solving / naming the overloads here :thinking: Or whether we always go to the most basic like QColor in this case (i think we have done this elsewhere)